### ISSUE TITLE
I have identified a core bug from clamscan init being called in parallel

### ISSUE THREAD

**Issue by niftylettuce** [kylefarris/clamscan#57]
It seems like this error occurs if clamscan init is being called in parallel, which would mean that the promise is not being shared, and false information being returned.

```sh
 Error (NodeClamError) {
    data: {},
    date: Date 2020-08-12 01:02:33 797ms UTC {},
    message: 'No valid & active virus scanning binaries are active and available and no host/socket option provided!',
  }
```

https://travis-ci.com/github/spamscanner/spamscanner/builds/179356176

Related to #55

**Comment by kylefarris** [comment]
Are you calling the `init` method in a `Promise.all()`? If so, why?

**Comment by kylefarris** [comment]
Looking at your code, it seems like you're calling `init` outside of the `Promise.all()` so I'm not entirely sure what exactly the issue is. I'd be glad to help if I knew a bit more about what's actually happening. Sorry if I'm misunderstanding.

**Comment by kylefarris** [comment]
I've patched the module. The patch is available in v1.3.3. Hopefully this fixes it for you.

**Comment by niftylettuce** [comment]
Thank you, I'll try it out
